### ISSUE TITLE
Several failures in FediTest Results

### ISSUE THREAD

**Issue by TomCasavant** [ckolderup/postmarks#197]
I didn't know the feditest team was planning on running tests on all the ActivityPub software so this is pretty cool. I think a lot of these are probably fixed by the changes @andypiper made to the content-type in #191 and I bet this will also improve compatibility in #124 

https://feditest.org/blog/2024-06-05-early-results-webfinger/
Matrix of test results: https://feditest.org/assets/2024-06-05/webfinger-server-all-wellknown-saas-imp.testmatrix.html
Detailed Test Results: https://feditest.org/assets/2024-06-05/webfinger-server-all-wellknown-saas-imp.sequential.html

- [ ]  webfinger.server.4_2__2_perform_query::normal_query Perform a normal, simple query on an existing account. - _Wrong HTTP content type._
- [ ] webfinger.server.4_2__4_do_not_accept_malformed_resource_parameters::double_equals_http_status Do not accept malformed resource parameters. Test HTTP status for inserting an extra = character. - _Not HTTP status 4xx._
- [ ] webfinger.server.4_2__4_do_not_accept_malformed_resource_parameters::double_equals_jrd Do not accept malformed resource parameters. Test JRD content for inserting an extra = character. - _Returns JRD content._
- [ ] webfinger.server.4_2__5_status_404_for_nonexisting_resources::status_404_for_nonexisting_resources The server responds with 404 when the resource parameter identifies a non-existent resource. - _Not HTTP status 404._
- [ ] webfinger.server.4_2__9_content_type::returns_jrd_in_response_to_https Test that a query over HTTPS produces a JRD. - _Wrong content type._
- [ ] webfinger.server.4_5__1_any_uri_scheme_for_resource_identifiers::any_uri_scheme_for_resource_identifiers The server must accept resource identifiers provided in the query that use any scheme. - _Not HTTP status 404._

**Comment by andypiper** [comment]
Would be great to get the Glitch server updated to the state of current main.

**Comment by TomCasavant** [comment]
> Would be great to get the Glitch server updated to the state of current main. 

Yeah, not sure when it was last updated (I think everything is still pretty much marked 0.0.1)

**Comment by ckolderup** [comment]
I also did not realize this was going to be run against Postmarks, but it's nice to see we passed _some_ of it, haha.

I did do some updates to the demo server on Glitch for a while but I can try to get it updated soon. I'm in the process of moving homes this spring/summer so I haven't been able to keep up with stuff much lately. Once I get into the new place and settled a bit I will try to make some time to get my Github notifications under control!
